sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -45,7 +45,8 @@ def _get_deferred_args(*args):
 
 
 class DeferredFrameTest(unittest.TestCase):
-  def _run_error_test(self, func, *args):
+  def _run_error_test(
+      self, func, *args, execution_time=False, distributed=True):
```

Comment:
`execution_time` feels a bit vague to me. What if we called it `construction_time` instead which is less ambiguous?

Reply:
Done